android/app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,8 @@
 
     <application
       android:supportsRtl="false"
+      android:largeHeap="true"
+      android:hardwareAccelerated="false"
```

Comment:
Just to make sure I understand correctly, why are we disabling hardware acceleration? The proposal [here](https://github.com/Expensify/App/issues/12893#issuecomment-1446422809) says:

> Displaying images larger than 8k pixels will also require enabling hardware acceleration and a large heap.

Reply:
@francoisl I made a mistake in the proposal. With enabled `hardwareAccelerated` together with `largeHeap` app won't display larger images over 8k pixels. 

Here are details of the native error `java.lang.RuntimeException: Canvas: trying to draw too large(394141296bytes) bitmap.`

Reply:
Got it, thanks for clarifying.